### PR TITLE
WIP: test jupyter cell include

### PR DESCRIPTION
**Pull request by natke**
Externally referenced repo is: https://github.com/microsoft/quantum
Path to external repo is relative to the docfx.json file i.e. `articles/quantum`

To reference this repo is ~/quantum

Screen Shot 2020-03-25 at 2 00 12 PM

[AB#1698154](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1698154)

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/libraries/standard/algorithms.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e24426420e2156db03fe2c4d44d28d5d4cfb6f9c/articles/libraries/standard/algorithms.md) | 73 | 56 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ef7e6dc-cd70-45f0-8165-b5c417149360_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/libraries/standard/algorithms.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/c058e68dd2a6c3230981966f8bcf225f6c6bab48/articles/libraries/standard/algorithms.md) | 73 | 56 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c09ab763-a8f2-4830-8ebb-9629a02653c6_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by cgranade** [comment]
Oh, this is neat, @natke! Thanks for working this out, it's exciting to be able to use our IQ# Notebook samples in docs this way! If I could ask, is it possible for cells to appear with Q# highlighting instead of Python highlighting? Thanks!

**Comment by natke** [comment]
That's a good question @cgranade. Let me find out.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/libraries/standard/algorithms.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0417c581848d8e4abc20d4138286ea2450b9ca07/articles/libraries/standard/algorithms.md) | 73 | 56 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/01970787-7df9-46c4-babd-f3bbfd6873e4_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by natke** [comment]
> Oh, this is neat, @natke! Thanks for working this out, it's exciting to be able to use our IQ# Notebook samples in docs this way! If I could ask, is it possible for cells to appear with Q# highlighting instead of Python highlighting? Thanks!

@cgranade Yes it is! Fixed.

File: articles/libraries/standard/algorithms.md [hunk 1]
```diff
@@ -60,6 +60,11 @@ Just as arithmetic plays a central role in classical computing, it is also indis
 $$
 \operatorname{Add}(b)\ket{a}=\ket{a + b}.
 $$
+
+Here is some code that shows the usage of the add operation.
```
Comment (cgranade) [inline articles/libraries/standard/algorithms.md]:
```suggestion
Here is some code that shows the usage of the `Add` operation.
```

**Comment by natke** [comment]
I wasn't planning on merging this PR @cgranade - it was rather a proof of concept. We can talk about where to formally include the code snippets from Jupyter notebooks

**Pull request closed without merging**